CMSgov/mdct-mcr
# MDCT-1960 pt. 2 - Archive frontend

## Description
[MDCT-1960](https://qmacbis.atlassian.net/browse/MDCT-1960)

### How to test
1. Login as a state user and create a report
2. Login as an admin user and go to that state's dashboard
3. Test the archive/unarchive button! It should 
- Change text after loading (from "Archive" to "Unarchive" or vice versa)
- Gray out the "Enter" button when archived
- Changed the `Status` text to "Archived" when archived, and the original status when unarchived
- When archived, if you switch back to state user, the report should NOT display

You can also use the api call directly while logged in as a state user to flip the state and see it display/not display (with refresh)
Try out the [postman collection from Confluence](https://qmacbis.atlassian.net/l/cp/1Hp1RQSg)

## Code author checklist

## Reviewer checklist (two different people)
